README.rst

```diff
@@ -62,5 +62,5 @@ SCICO is distributed as open-source software under a BSD 3-Clause License (see t
 
 LANL open source approval reference C20091.
 
-(c) 2020-2021. Triad National Security, LLC. All rights reserved.
+(c) 2020-2022. Triad National Security, LLC. All rights reserved.
```

Comment:
While you are at it, `overview.rst` also needs that change.

Reply:
Similarly,

`scico/_autograd.py`

`scico/diagnostics.py`

`scico/plot.py`

`scico/linop/_matrix.py`

`scico/linop/_stack.py`

Reply:
Noted on `overview.rst`. The policy on the code is to bump the copyright year the first time a module is edited in a new calendar year. Have any of these been touched this year?

Reply:
I see. The other ones are all >= 2 months old.